src/file.cpp

```diff
@@ -336,11 +336,11 @@ static_assert((libtorrent::file::sparse & libtorrent::file::attribute_mask) == 0
 
 namespace libtorrent
 {
-	int bufs_size(file::iovec_t const* bufs, int num_bufs)
+	int bufs_size(span<file::iovec_t const> bufs)
 	{
 		std::size_t size = 0;
-		for (file::iovec_t const* i = bufs, *end(bufs + num_bufs); i < end; ++i)
-			size += i->iov_len;
+		for (auto buf : bufs)
```

Comment:
In general I prefer to handle spans by value. I only use auto& when I need to modify the entries.